src/util/rocksdb/mod.rs

```diff
@@ -80,14 +80,19 @@ impl<'a> CFOptions<'a> {
     }
 }
 
-pub fn new_engine(path: &str, cfs: &[&str]) -> Result<DB, String> {
+pub fn new_engine(path: &str, cfs: &[&str], opts: Option<Vec<CFOptions>>) -> Result<DB, String> {
```

Comment:
can we pass `Option<&[CFOptions]>`

Reply:
I think it doesn't matter now, because the caller don't need the options.

Reply:
should keep the same style with cfs.